_drafts/making-backend-scalable.markdown

```diff
@@ -0,0 +1,123 @@
+---
+layout: post
+title: "Making Autolab's Backend Scalable"
+author: Ilter Canberk
+
+tags: [tango, redis, scalability]
+
+---
+
+Tango is the back-end service that receives grading jobs from Autolab's front-end, puts them in a job queue, and assigns them to available containers for grading. Tango came across larger work-loads and huge files as Autolab got more popular this semester.
+
```

Comment:
GP. I will put an intro to Tango but I think it can be shorter and we can link to Tango repo. (I also realized that Tango Readme should include some technical overview too)

(Slideshare link is legit)